[Clock] Fix `ClockAwareTrait` usage (#17772)
symfony/symfony-docs: components/clock.rst

to any arbitrary date/time.

In order to use this component in your services, make their classes use the

> :class:`Symfony\\Component\\Clock\\ClockAwareTrait`. This trait defines the
> ``ClockAwareTrait::setClock()`` method. Thanks to the ``#[Required]`` attribute
> declared on this method and if your application uses :ref:`service autoconfiguration <services-autoconfigure>`,
> ``ClockAwareTrait::setClock()`` will automatically be called by the service container,
> injecting a clock implementation into the ``$clock`` property (also provided
> by the trait). Your services can now call the ``$this->now()`` method to get the
> current time::

Reviewer: ```suggestion
:class:`Symfony\\Component\\Clock\\ClockAwareTrait`. Thanks to
:ref:`service autoconfiguration <services-autoconfigure>`, the ``setClock()`` method
of the trait will automatically be called by the service container.

You can now call the ``$this->now()`` method to get the current time::
```
